files/galaxy/dynamic_job_rules/production/total_perspective_vortex/tools.yml

```diff
@@ -3203,6 +3203,12 @@ tools:
       require:
       - pulsar
       - pulsar-qld-gpu
+  alphafold_test_2:
+    context:
+      alphafold_db: /data2
+    docker_volumes:
```

Comment:
changing docker_volumes on the tool won't make a difference because the destination overrides them.  Please change it here: https://github.com/usegalaxy-au/infrastructure/blob/master/files/galaxy/dynamic_job_rules/production/total_perspective_vortex/destinations.yml.j2#L346